transcom/mymove
# B 19827 int

## [B-19827](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A946538)

## Summary

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the admin app-
2. go to office users
3. see task invoicing officer instead of transportation invoicing officer
4. create a TIO - see that you are able to login successfully

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots

![image](https://github.com/transcom/mymove/assets/136509766/d934a753-787e-422b-8dff-572e6d35aed5)
![image](https://github.com/transcom/mymove/assets/136509766/e90d598c-693f-4c67-9614-48a0b661d3cd)
![image](https://github.com/transcom/mymove/assets/136509766/e9438bcc-8876-4a33-bbeb-ba9a86caad1e)
